contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```
@@ -517,6 +541,108 @@ resolve_numeric_typmod_outer_var(Plan *plan, AttrNumber attno)
 	return resolve_numeric_typmod_from_exp(outerplan, (Node *)tle->expr);
 }
 
+static bool is_tsql_bit_numeric(Oid oid)
+{
+	if (tsql_bit_numeric_oid == InvalidOid)
+	{
+		Oid			nspoid;
+		Oid			typoid;
+		Oid 		funcargtypes[1];
+
+		nspoid = get_namespace_oid("sys", true);
```

we should keep the second parameter as `false` which will ensure that if `get_namespace_oid` function is unable to find oid of `sys` then we should throw error.

---

changed the parameter to false so as we can throw error if oid is not found.